flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/PipelinedSubpartitionWithReadViewTest.java

```diff
@@ -369,7 +388,7 @@ static void assertNextBuffer(
 				null,
 				expectedIsMoreAvailable,
 				expectedBuffersInBacklog,
-				expectedNextBufferIsEvent,
+				expectedIsMoreAvailableWithoutCredit,
```

Comment:
Unify the same naming for `expectedIsMoreAvailableWithoutCredit` as below `expectedIsEventAvailable`? also for `expectedIsMoreAvailable`